app/src/main/java/org/exoplatform/model/SocialActivity.java

```diff
@@ -42,7 +45,7 @@
 
   public final static String TYPE_DEFAULT_SPACE = "exosocial:spaces";
 
-  public final static String TYPE_DOC           = "DOC_ACTIVITY";
+  public static String       TYPE_DOC           = "files:spaces";
```

Comment:
Don't change this constant to a static variable.

You can add constants:
```java
  public final static String       OLD_DOC_ACTIVITY_TYPE           = "files:spaces";

  public final static String       DOC_ACTIVITY_TYPE                     = "DOC_ACTIVITY";

```

Reply:
done